src/core/core.scale.js

```diff
@@ -157,11 +159,13 @@ function computeLabelSizes(ctx, tickFonts, ticks, caches) {
 		widths.push(width);
 		heights.push(height);
 		offsets.push(lineHeight / 2);
+		widestLabelSize = Math.max(width, widestLabelSize);
+		highestLabelSize = Math.max(height, highestLabelSize);
 	}
 	garbageCollect(caches, length);
 
-	widest = widths.indexOf(Math.max.apply(null, widths));
-	highest = heights.indexOf(Math.max.apply(null, heights));
+	widest = widths.indexOf(widestLabelSize);
```

Comment:
More of a question, if these arrays are quite long, should the widest & highest indexes be tracked in the initial loop too?

Reply:
That's a good point. I stopped once the example in the issue worked, but that would likely improve the performance.

Reply:
I did some testing, and this didn't seem to make a noticeable difference, so I left it as-is.